### DISCUSSION TITLE
Hosting with a custom server.jar (for BTA)

### DISCUSSION THREAD

**Discussion by Lunatunny** [itzg/docker-minecraft-server#2719] [Q&A]
Hello! I want to play a mod called "Better Than Adventure" for 1.7.3, except instead of providing mods for the server, it provides a custom server.jar.

Is there any way I can like, mount this into the container and prevent the automation from replacing/updating it? Or something, anything.

**Comment by itzg** [comment]
https://docker-minecraft-server.readthedocs.io/en/latest/types-and-platforms/server-types/others/#custom

**Reply by Lunatunny** [reply]
Thank you! Sorry I missed this thanks for your patience.
